pages/builders-challenge/index.html

```diff
@@ -239,11 +282,32 @@ <h3>Our Partners</h3>
     </div>
 </section>
 
+<section class="mzp-l-content mzp-t-content-xl c-challenge-event">
+    <table id="challenge-schedule" class="mzp-u-data-table">
+        <thead>
+            <tr>
+                <img src="/img/builders-challenge/event-agenda.png" alt="Event Agenda">
```

Comment:
I'd say this should go outside the table, and probably be wrapped in an `h3`. Might need a little CSS to center it and fix any spacing.

OR if you want it to be part of the table it could be in a `caption` rather than `thead`